python/main.py

```diff
@@ -55,27 +60,46 @@ async def lifespan(app: FastAPI):
 class HelloResponse(BaseModel):
     message: str
 
+@app.get("/items") 
+def get_items():
+    with open('items.json', 'r') as json_open:
```

Comment:
If `items.json` doesn't exist in server, we should return some default value probably